napari/__init__.pyi

```diff
@@ -8,6 +8,7 @@ __version__: str
 
 notification_manager: napari.utils.notifications.NotificationManager
 Viewer = napari.viewer.Viewer
+current_viewer = Callable[[], Viewer]
```

Comment:
nope!  the `=` sign here means it's now a type alias (i.e. "current_viewer" is an alias for the type hint `Callable[]`).  You need to use a colon if you're trying to say that current_viewer is a Callable that returns a viewer

Reply:
(or, use an equals sign and make it equal to napari.viewer.current_viewer)

Reply:
oh dear, thanks! Fixing :)

Reply:
should've known better than to doubt you 🤣